spec/features/authentication/reset_password.rb

```diff
@@ -0,0 +1,28 @@
+require 'feature_helper'
+
+feature 'reset password', js: true do
+  given!(:user) { create :user }
+
+  Steps 'for requesting reset instructions' do
+    When 'I visit the homepage' do
+      visit '/'
+    end
+    And 'I click on Login' do
+      within '.nav' do
+        click_link 'Login'
+      end
+    end
+    And 'I ask for reset instructions' do
+      click_link 'Forgot your password?'
+    end
+    And 'I fill in my username' do
+      fill_in 'Username', with: user.username
+      click_on 'Send me reset password instructions'
+    end
+    Then 'An email should be sent' do
+      sleep 1
```

Comment:
Instead of using the sleep, you can do

should_see 'text for success flash message'

It will wait for a few seconds to see that message.  Then after that line, you know the request is done.

It will also take much less than one second.